command/kv_metadata_put.go

```diff
@@ -71,14 +71,13 @@ func (c *KVMetadataPutCommand) Flags() *FlagSets {
 	f.IntVar(&IntVar{
 		Name:    "max-versions",
 		Target:  &c.flagMaxVersions,
-		Default: 0,
+		Default: -1,
 		Usage:   `The number of versions to keep. If not set, the backend’s configured max version is used.`,
 	})
 
-	f.BoolVar(&BoolVar{
+	f.BoolPtrVar(&BoolPtrVar{
```

Comment:
I don't quite understand why `BoolPtrVar` is being used here over `BoolVar`, but I also don't really understand the purpose of `BoolVar` versus just using the bool. Could you clarify what the purpose of these are?

Reply:
Using `BoolPtrVar` allows us to determine whether or not the flag was actually specified.